doc/rst/usingchapel/chplenv.rst

```diff
@@ -754,9 +754,11 @@ CHPL_LLVM
        Value          Description
        ============== ======================================================
        bundled        use the llvm/clang distribution in third-party
-       system         find a compatible LLVM in system libraries;
-                      note: the LLVM must be a version supported by Chapel
+       system         find a compatible LLVM and clang in system libraries;
+                      but note that it must be a version supported by Chapel
        none           do not support llvm/clang-related features
+                      (but note that the LLVM Support library will still
+                      be used -- see ``CHPL_LLVM_SUPPORT`` below)
```

Comment:
I think this could be a link to the section below, but it's okay to leave it as is.

Reply:
Yeah I was worried the link in the table would be visually distracting for somebody reading the .rst file directly. I'll leave this one as is.